api_tests/dry/multiple_peers.js

```diff
@@ -15,15 +16,21 @@ const alpha_asset_bob_quantity = "100000000";
 const alpha_asset_charlie_quantity = "200000000";
 
 const beta_asset_name = "Ether";
-const beta_asset_bob_quantity = Web3.utils.toWei("10", "ether");
-const beta_asset_charlie_quantity = Web3.utils.toWei("20", "ether");
+const beta_asset_bob_quantity = utils.toWei("10", "ether");
+const beta_asset_charlie_quantity = utils.toWei("20", "ether");
 
 const alpha_expiry = new Date("2080-06-11T23:00:00Z").getTime() / 1000;
 const beta_expiry = new Date("2080-06-11T13:00:00Z").getTime() / 1000;
 
-const alice = actor.create("alice");
-const bob = actor.create("bob");
-const charlie = actor.create("charlie");
+const alice = actor.create("alice", {
+    ethConfig: global.harness.ledgers_config.ethereum,
```

Comment:
:question: Why do the actors not need a `bitcoin` config?

Reply:
Check the new todo in `actor.js`.

This will be cleaned up with a future PR when migrating actor to TypeScript

Reply:
What's the reason not to use global configs in the bitcoin and ethereum files?
What's your "clean up" plan with regards to the rpc config?

Reply:
I believe that the libraries should be passed the values instead of using a global variable.